CONTRIBUTING.md

```diff
@@ -1,98 +1,26 @@
-# Questions
```

Comment:
I'd maybe add a note about this being an adopted-ember-addon and link to the doc explaining what that means for contributing

Reply:
Is there a recommended CONTRIBUTING.md for adopted packages?